reference-data/build.gradle

```diff
@@ -1,5 +1,32 @@
 description = 'Managing reference data required for payments'
+configurations.all {
```

Comment:
As discussed & shown please keep the config in Root Build.gradle [all projects section]